pkg/scheduler/scheduler.go

```diff
@@ -212,7 +212,9 @@ func (s *Scheduler) FrontendLoop(ctx context.Context, frontend *connect.BidiStre
 	if err != nil {
 		return err
 	}
-	defer s.frontendDisconnected(frontendAddress)
+	defer func() {
+		s.frontendDisconnected(frontendAddress)
+	}()
```

Comment:
Nit:  anonymous func()` likely from debugging, not really necessary anymore

Reply:
true :)